src/defaultsettings.cpp

```diff
@@ -293,6 +293,7 @@ void set_default_settings()
 	// Input
 	settings->setDefault("invert_mouse", "false");
 	settings->setDefault("mouse_sensitivity", "0.2");
+	settings->setDefault("ads_sensitivity", "1.0");
```

Comment:
why not call it mouse_ads_sensitivity if it's related to mouse ? i don't know about the term ADS and never saw it on various AAA games, maybe there is better UX wording ?

Reply:
this setting affects both the mouse and joystick, and isn't really a concept specific to any particular device. if they have the fov sensitivity adjustment already in the code, then it can now be controlled with this setting.

it's "ads" for aim down sites, it's the only term for this that's short and common. if you can come up with a better word for it you can say but i'm not sure there really is one